custom_components/plugwise-beta/__init__.py

```diff
@@ -36,9 +37,15 @@ async def async_setup_entry(hass: HomeAssistant, entry: ConfigEntry) -> bool:
     )
 
     try:
-        await api.connect()
+        connected = await api.connect()
+
+        if not connected:
+            _LOGGER.error("Unable to Smile: %s",api.smile_status)
```

Comment:
Unable to connect to Smile? Seems a word is missing.

Reply:
fixed